src/status_im/ui/screens/chat/components/input.cljs

```diff
@@ -554,30 +555,30 @@
                           :pdy   0      ;used for gesture
                           :state :min   ;:min, :custom-chat-available, :custom-chat-unavailable, :max
                           :clear false})
-           keyboard-was-shown (atom false)]
+           keyboard-was-shown (atom false)
+           text-input-ref (quo.react/create-ref)
+           send-ref (quo.react/create-ref)
+           refs {:send-ref       send-ref
+                 :text-input-ref text-input-ref}]
        (fn []
          [:f>
           (fn []
-            (let [text-input-ref (quo.react/create-ref)
-                  send-ref (quo.react/create-ref)
-                  refs {:send-ref       send-ref
-                        :text-input-ref text-input-ref}
-
+            (let [reply @(re-frame/subscribe [:chats/reply-message])
```

Comment:
@flexsurfer should we be using `>sub` here instead? 🤔 
i.e from `[status-im.utils.handlers :refer [<sub ]]`

Reply:
yes, thank you